prober/http.go

```diff
@@ -406,7 +412,8 @@ func ProbeHTTP(ctx context.Context, target string, module config.Module, registr
 		}
 
 		if resp != nil && !requestErrored {
-			_, err = io.Copy(ioutil.Discard, resp.Body)
+			buf := &bytes.Buffer{}
+			respBodyBytes, err = io.Copy(buf, resp.Body)
```

Comment:
Why not keep using ioutil.Discard?

Reply:
You're right. I misunderstood the documentation.